Scripts/Bootstrap.ps1

```diff
@@ -0,0 +1,81 @@
+﻿param (
+    [Parameter(Mandatory=$true)]
+    [string]$NuGetDropPath,
+    [Parameter(Mandatory=$true)]
+    [string]$FuncTestRoot)
```

Comment:
I'd suggest having a default value for the test root, smth. like ${env:USERPROFILE}\...\FuncTest